src/javaharness/java/arcs/api/SingletonProxy.java

```diff
@@ -1,29 +1,29 @@
-package arcs.api;
+// package arcs.api;
```

Comment:
Why is this commended out?

Reply:
Shane and I agreed to temporarily only use Collection handles in Java PEC until the new storage rolls out (because the old storage single proxy is hard to express with new CRDTs)